Move existing examples to subfolder

lola-masson opened the pull request:
in prevision of ovp8xx-based examples

lola-masson committed:
Move existing examples to subfolder

graugans commented:
Some suggestions:

- Maybe it is a good idea to have some general information about compatibility.  For example that in case one is looking for examples which support the legacy branch of ifm3d. We may also provide an legacy examples branch?
- The LICENSE file is common for all examples and shall not be moved
- The file `o3d3xx-o3x1xx/clean_emacs.sh` can be removed IMHO, maybe a second step

lola-masson committed:
Move license file to parent folder

lola-masson commented:
Yes, you are right. Currently, I have not changed anything from what was already there. I have added your suggestion to internal issue #265. If that's OK with you I'd delay the suggested changes until a full update of the existing examples.

I moved the license file to parent folder.
